src/swift/Source.swift

```diff
@@ -124,8 +124,17 @@ extension DispatchSource {
 		public static let exit = ProcessEvent(rawValue: 0x80000000)
 		public static let fork = ProcessEvent(rawValue: 0x40000000)
 		public static let exec = ProcessEvent(rawValue: 0x20000000)
+#if os(FreeBSD)
+		public static let track = ProcessEvent(rawValue: 0x00000001)
```

Comment:
Why the different name? Should we not be using the name names to maintain source consistency?

Reply:
This is because `NOTE_TRACK` is only available on FreeBSD but not Darwin

Reply:
Sure, having an extended FreeBSD only option is fine, but this does remove the `signal` note doesn't it? I take it that `NOTE_TRACK` is meant to replace that? If so, why not keep the name incongruous with the underlying note to allow easier porting?

Reply:
Sorry, I should've add more context in my previous reply. `NOTE_SIGNAL` on Darwin and `NOTE_TRACK` on FreeBSD are not interchangeable and each are platform specific.

`NOTE_SIGNAL` on Darwin allow users to check if the process was sent a signal,

on the other hand, `NOTE_TRACK` on FreeBSD cause kqueue to follow the target process across fork and automatically register events with same monitor. For example, `NOTE_TRACK | NOTE_EXIT` on a pid cause exit event of the tracing process, and all its decedents to be reported.